src/cli.js

```diff
@@ -130,12 +144,16 @@ program
   .argument("[folder]", "convert by folder")
   .action(
     tryOrExit(async ({ args, options }) => {
-      const folder = args.folder || "";
+      let folder = (args.folder || "").replace(/\\|\//g, path.sep); // correct path separator
+      if (folder.length !== 0 && !folder.endsWith(path.sep)) { // if folder is specified, find folder only
+        folder += path.sep;
```

Comment:
`Document.findAll` will metch the filename, such as `web/api/index.html`, add a tailing slash to match folder